c/json.c

```diff
@@ -953,6 +957,25 @@ void jsonParseFail(JsonParser *parser, char *formatString, ...) {
   }
 }
 
+static int
+jsonTokenizerGrowBuffer(JsonTokenizer *t) {
+  if (t->bufferSize >= INT32_MAX / 2) {
+    return -1;
+  }
+  int newSize = 2 * t->bufferSize;
+  if (newSize >= JSON_TOKEN_BUFFER_SIZE_LIMIT) {
+    return -1;
+  }
+  void *newBuffer = safeRealloc(t->buffer, newSize,
```

Comment:
Surprisingly safeRealloc is within this file. It seems to be completely generic so it should be moved to alloc.c

Reply:
done